### PR TITLE
add screenshot

### PR DESCRIPTION
**Pull request by ayushmunot**

### CODE REVIEW

**Comment by ayushmunot** [comment]
added screenshot

**Comment by diversys** [comment]
Is this just a random screenshot?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Filer
```
Comment (humdingerb) [inline README.md]: While you're touching the readme, use markdown for formatting. There are markdown editors online.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+![Filer App](https://github.com/ayushmunot/Filer/blob/master/filer1.png?raw=true "filer")
```
Comment (humdingerb) [inline README.md]: That's not a screenshot of Filer.
For the real screenshot, show Filer's main window and a "Edit rule" window nicely positioned. You'll have to take a screenshot of a full screen and crop etc. in an image editor (e.g. Wonderbrush). The background has to be transparent.
Join the IRC channel #haiku if you need help.

**Comment by scottmc** [comment]
Looks like it has a screenshot now, so closing this one out.

**Pull request closed without merging**